### PR TITLE
Bounced Invitations

### PR DESCRIPTION
**Pull request by proteusvacuum**
https://trello.com/c/sWyqlaAc/133-make-bounced-invites-visible-per-project-space

##### SUMMARY

This adds delivery information to the invites page.
Caveat is that most mail providers will show a message as "delivered", but then will change it to "bounced" later if the message then fails to send.

Screenshot from 2020-08-19 14-46-39

FYI @dimagi/product

### CODE REVIEW

File: corehq/apps/users/static/users/js/web_users.js [hunk 1]
```diff
@@ -169,16 +169,27 @@ hqDefine("users/js/web_users",[
...
+        self.allStatuses = _.uniq(_.map(self.allInvitations(), function (i) {
+            return i.email_status;
+        }));
         self.itemsPerPage = ko.observable();
         self.totalItems = ko.observable(self.allInvitations().length);
         self.showPagination = ko.computed(function () {
             return self.totalItems() > self.itemsPerPage();
         });
         self.goToPage = function (page) {
+            console.log('hello');
```
Comment (biyeun) [inline corehq/apps/users/static/users/js/web_users.js:182]: console.log

Comment (proteusvacuum) [reply corehq/apps/users/static/users/js/web_users.js:182]: oops, thanks

**Review by biyeun — APPROVED** [review]
small nitpicks from me and stickler but otherwise looks great. thanks!

**Comment by orangejenny** [comment]
Nice! I haven't read the code, just have a UI nitpick: I don't think the labels are necessary. `Search by email address` could be the placeholder in the search box, and the statuses seem self explanatory.

**Comment by proteusvacuum** [comment]
@orangejenny I somewhat agree... But what do you do for the default when the page loads in that case? e.g. https://staging.commcarehq.org/a/farid-remote-test/settings/users/web/ 
I currently have it say "All", which is not very self-explanatory. I guess I could change it to say "Filter by status..." ?

**Comment by orangejenny** [comment]
What if it said "All Statuses" or "Any Status"?

**Comment by orangejenny** [comment]
Hey @proteusvacuum , this looks like it's causing https://sentry.io/organizations/dimagi/issues/1860535486/ because it's passing a string as the messaging event id, but `mark_subevent_bounced` expects that to be a numeric id. Could you take a look, or let me know if you don't have time and USH should pick it up?

**Pull request merged**